src/Components/Table/DjTable.tsx

```diff
@@ -164,23 +181,41 @@ export default class DjTable extends React.Component<
     let apiUrl =
       `${process.env.REACT_APP_DJSCIVIZ_BACKEND_PREFIX}` + this.props.route
 
-    if(!this.state.sorter.join(",").includes('ASC') && !this.state.sorter.join(",").includes('DESC')){
-      apiUrl = apiUrl + '?page=' + this.state.offset + '&limit=' + this.state.numberOfTuples;
-    } else { 
-      apiUrl = apiUrl + '?page=' + this.state.offset + '&limit=' + this.state.numberOfTuples + '&order=' + this.state.sorter.join(",");
+    if (
+      !this.state.sorter.join(',').includes('ASC') &&
+      !this.state.sorter.join(',').includes('DESC')
+    ) {
+      apiUrl =
+        apiUrl +
+        '?page=' +
+        this.state.offset +
+        '&limit=' +
+        this.state.numberOfTuples
+    } else {
+      apiUrl =
+        apiUrl +
+        '?page=' +
+        this.state.offset +
+        '&limit=' +
+        this.state.numberOfTuples +
+        '&order=' +
+        this.state.sorter.join(',')
     }
 
-    if(this.props.restrictionList.length > 0){
+    if (
+      this.props.restrictionList.length >= 1 &&
+      this.props.restrictionList[0] != ''
+    ) {
       apiUrl = apiUrl + '&' + this.props.restrictionList.join('&')
     }
 
-    apiUrl = apiUrl.concat("&")
+    if (queryParamList.length > 0) {
```

Comment:
`length` has a truthy value, no?

```suggestion
    if (queryParamList.length) {
```